modules/tinymce/changelog.txt

```diff
@@ -7,6 +7,7 @@ Version 5.7.0 (TBD)
     Added new `font_css` setting for fonts to be added to both containing document and the editor #TINY-6199
     Added a new `ImageUploader` API to simplify uploading image data to the configured `images_upload_url` or `images_upload_handler` #TINY-4601
     Added Oxide variable to define the container background color in fullscreen mode #TINY-6903
+    Added enhanced table resizing behavior when inserting or deleting columns with improved respect for the `table_column_resizing` option #TINY-6711
```

Comment:
This reads very strangely to me, but I can't think of any suggestions atm :cry:

Reply:
NTS: Split into a Bugfix and a Change